packages/venia-ui/lib/components/Button/button.css

```diff
@@ -34,7 +34,8 @@
     --stroke: var(--venia-brand-color-1-800);
 }
 
-.root:disabled {
+.root:disabled,
+.root:hover:disabled {
```

Comment:
Did you have a specific browser or use case that required the second line here? If not, can we remove it? It looks redundant, and a quick test in Chrome & Firefox seems to bear that out.

Reply:
@jimbo chrome keeps the `:hover` state on the button once you press it. Safari does something similarly weird where you have to move the mouse by at least 1 pixel to remove the `:hover` state. This caused a button which became disabled to remain active until something removed the hover state.

Having the `:disabled` state override the `:hover` state seemed the easier solution here. There might be an underlying problem here with the hover states that I'm unaware of, so if there's a better way to resolve this please let me know!

Reply:
No, that's enough of a reason to keep it as-is. If you add a comment, I'll approve this. 👍

Reply:
@jimbo comment added 😄